app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -564,16 +567,24 @@ class BrowserRobot {
         )
     }
 
-    fun verifyPrefilledLoginCredentials(userName: String, password: String) {
+    fun verifyPrefilledLoginCredentials(userName: String, password: String, credentialsArePrefilled: Boolean) {
         var currentTries = 0
```

Comment:
you can simplify this using the for (i in 1..RETRY_COUNT) {...} format

Reply:
Done, thanks !